src/codegen/codegen_cuda.cc

```diff
@@ -90,7 +90,7 @@ void CodeGenCUDA::PrintType(Type t, std::ostream& os) {  // NOLINT(*)
         if (t.lanes() == 4) {
           // directly 4 8 bit int in integer.
           enable_int8_ = true;
-          os << "char4"; return;
+          os << "int"; return;
```

Comment:
Please add a comment block here on why are we making this choice, so people won't change it back